[frontend]MKMediaVideo.vue Broken

### 💡 概要

```
 => ERROR [web native-builder 45/45] RUN pnpm build                                                                                                                                    47.0s
------
 > [web native-builder 45/45] RUN pnpm build:
3.548
3.548 > misskey@2024.10.1-taiyme.1-SHINANOMAI.2 build /misskey
3.548 > pnpm build-pre && pnpm -r build && pnpm build-assets
3.548
4.409
4.409 > misskey@2024.10.1-taiyme.1-SHINANOMAI.2 build-pre /misskey
4.409 > node ./scripts/build-pre.js
4.409
5.566 Scope: 8 of 9 workspace projects
5.568 packages/misskey-js build$ node ./build.js
5.569 packages/misskey-reversi build$ node ./build.js
5.570 packages/misskey-bubble-game build$ node ./build.js
6.417 packages/misskey-bubble-game build: [misskey-bubble-game] start building...
6.418 packages/misskey-reversi build: [misskey-reversi] start building...
6.419 packages/misskey-js build: [misskey-js] start building...
6.921 packages/misskey-reversi build: [misskey-reversi] build succeeded.
6.921 packages/misskey-reversi build: [misskey-reversi] skip building d.ts because NODE_ENV is production.
6.921 packages/misskey-reversi build: [misskey-reversi] finish building.
6.922 packages/misskey-bubble-game build: [misskey-bubble-game] build succeeded.
6.922 packages/misskey-bubble-game build: [misskey-bubble-game] skip building d.ts because NODE_ENV is production.
6.922 packages/misskey-bubble-game build: [misskey-bubble-game] finish building.
6.929 packages/misskey-js build: [misskey-js] build succeeded.
6.929 packages/misskey-js build: [misskey-js] skip building d.ts because NODE_ENV is production.
6.930 packages/misskey-js build: [misskey-js] finish building.
6.930 packages/misskey-reversi build: Done
6.932 packages/misskey-bubble-game build: Done
6.933 packages/misskey-js build: Done
6.933 packages/backend build$ swc src -d built -D --strip-leading-paths
6.934 packages/frontend-shared build$ node ./build.js
6.934 packages/sw build$ node build.js
7.138 packages/sw build: (node:134) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
7.139 packages/sw build: (Use `node --trace-warnings ...` to show where the warning was created)
7.179 packages/frontend-shared build: [frontend-shared] start building...
7.252 packages/frontend-shared build: [frontend-shared] build succeeded.
7.253 packages/frontend-shared build: [frontend-shared] skip building d.ts because NODE_ENV is production.
7.274 packages/frontend-shared build: [frontend-shared] finish building.
7.279 packages/frontend-shared build: Done
8.544 packages/sw build: Starting SW building...
8.868 packages/sw build: done
8.877 packages/sw build: Done
14.69 packages/backend build: Successfully compiled: 847 files, copied 29 files with swc (612.06ms)
14.70 packages/backend build: Done
14.70 packages/frontend build$ vite build
14.70 packages/frontend-embed build$ vite build
19.11 packages/frontend build: vite v5.4.8 building for production...
19.11 packages/frontend-embed build: vite v5.4.8 building for production...
19.44 packages/frontend-embed build: transforming...
19.45 packages/frontend build: transforming...
27.06 packages/frontend-embed build: ✓ 215 modules transformed.
27.31 packages/frontend-embed build: rendering chunks...
27.45 packages/frontend-embed build: ../../built/_frontend_embed_vite_/test-webgl2-CmX4Kj3b.js        0.13 kB
27.45 packages/frontend-embed build: ../../built/_frontend_embed_vite_/manifest.json                  1.64 kB
27.45 packages/frontend-embed build: ../../built/_frontend_embed_vite_/draw-blurhash-Bp1JgrMM.js      3.55 kB
27.45 packages/frontend-embed build: ../../built/_frontend_embed_vite_/Cqq1W_WT.png                   6.29 kB
27.46 packages/frontend-embed build: ../../built/_frontend_embed_vite_/B8orLCnS.woff2               813.70 kB
27.46 packages/frontend-embed build: ../../built/_frontend_embed_vite_/CpvelmIk.woff              1,150.70 kB
27.46 packages/frontend-embed build: ../../built/_frontend_embed_vite_/CweI_gao.ttf               2,287.31 kB
27.46 packages/frontend-embed build: ../../built/_frontend_embed_vite_/7yDTwB9M.css                  23.42 kB
27.46 packages/frontend-embed build: ../../built/_frontend_embed_vite_/K30lDJ5t.css                 207.13 kB
27.47 packages/frontend-embed build: ../../built/_frontend_embed_vite_/app-BWH01IeP.js               29.31 kB
27.47 packages/frontend-embed build: ../../built/_frontend_embed_vite_/BTTihJsY.js                   69.01 kB
27.47 packages/frontend-embed build: ../../built/_frontend_embed_vite_/DPgGzTpj.js                  127.12 kB
27.47 packages/frontend-embed build: ✓ built in 8.27s
27.49 packages/frontend-embed build: Done
44.43 packages/frontend build: ✓ 1821 modules transformed.
44.49 packages/frontend build: x Build failed in 25.25s
44.49 packages/frontend build: error during build:
44.49 packages/frontend build: [vite:css] [sass] expected "}".
44.49 packages/frontend build:     ╷
44.49 packages/frontend build: 299 │     }
44.49 packages/frontend build:     │      ^
44.49 packages/frontend build:     ╵
44.49 packages/frontend build:   src/components/MkMediaVideo.vue 299:3  root stylesheet
44.49 packages/frontend build: file: /misskey/packages/frontend/src/components/MkMediaVideo.vue?vue&type=style&index=0&lang.module.scss
44.49 packages/frontend build: Error: expected "}".
44.49 packages/frontend build:     ╷
44.49 packages/frontend build: 299 │     }
44.49 packages/frontend build:     │      ^
44.49 packages/frontend build:     ╵
44.49 packages/frontend build:   src/components/MkMediaVideo.vue 299:3  root stylesheet
44.49 packages/frontend build:     at Object.wrapException (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:2301:43)
44.49 packages/frontend build:     at SpanScanner.error$3$length$position (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:84065:15)
44.49 packages/frontend build:     at SpanScanner.error$1 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:84068:19)
44.49 packages/frontend build:     at ScssParser0._stylesheet0$_styleRule$2 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:118240:23)
44.49 packages/frontend build:     at ScssParser0._stylesheet0$_declarationOrStyleRule$0 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:118099:88)
44.49 packages/frontend build:     at ScssParser0._stylesheet0$_statement$1$root (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:117992:162)
44.49 packages/frontend build:     at ScssParser0._stylesheet0$_statement$0 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:117995:19)
44.49 packages/frontend build:     at tear_off.call$0 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:2621:45)
44.49 packages/frontend build:     at ScssParser0.children$1 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:113857:33)
44.49 packages/frontend build:     at ScssParser0._stylesheet0$_withChildren$1$3 (/misskey/node_modules/.pnpm/sass@1.79.3/node_modules/sass/sass.dart.js:121141:39)
44.96 packages/frontend build: Failed
44.98 /misskey/packages/frontend:
44.98  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  frontend@ build: `vite build`
44.98 Exit status 1
45.29  ELIFECYCLE  Command failed with exit code 1.
------
failed to solve: process "/bin/sh -c pnpm build" did not complete successfully: exit code: 1


```

### 🥰 予想される挙動

ビルドが正常に完了する

### 🤬 実際の挙動

ビルドが失敗する

### 📝 再現手順

1.git clone this repo
2.docker compose build
3.ERROR!!


### 💻 フロントエンドの環境

- Model and OS of the device(s):
- Browser:
- Server URL:
- Misskey:2024.10.1-taiyme.1-SHINANOMAI.2


### 🛰 バックエンドの環境 (サーバー管理者向け)

- Installation Method or Hosting Service:Ubuntu Server 24.04 KDE
- Misskey:2024.10.1-taiyme.1-SHINANOMAI.2
- Node:謎
- PostgreSQL:謎
- Redis:謎
- OS and Architecture:Ubuntu 24.04 KDE


### この問題は自分で解決したいですか？

- [ ] はい、自分でパッチを当ててプルリクエストを作成します

--- a/packages/frontend/src/components/MkMediaVideo.vue
+++ b/packages/frontend/src/components/MkMediaVideo.vue
@@ -807,4 +807,5 @@ const {
 			--mediaVideo-scale: 0.60;
 		}
 	}
-	</style>
+}
+</style>

